packages/common/test/restHelpers.test.ts

```diff
@@ -91,12 +1069,13 @@ describe("Module `restHelpers`: common functions involving the arcgis-rest-js li
           return response;
         }
       );
-      restHelpers
-        .createUniqueFolder(folderTitleRoot, MOCK_USER_SESSION)
-        .then((response: portal.IAddFolderResponse) => {
+      createUniqueFolder(folderTitleRoot, MOCK_USER_SESSION).then(
+        (response: portal.IAddFolderResponse) => {
           expect(response).toEqual(expectedSuccess);
           done();
-        }, done.fail);
+        },
+        done.fail
```

Comment:
I didn't think about doing it like this. I like it...even cleaner than `() => done.fail()`